src/index.js

```diff
@@ -11,6 +11,7 @@ import reportWebVitals from './reportWebVitals';
 ReactDOM.render(
   <React.StrictMode>
     <Header />
+    <Login />
     <App />
     <Form />
     <Profile />
```

Comment:
As we suggested in the last code review, please add these to the `App.js` and not here in the `index.js`.